modules/microshift-rhoai-verify-model-connected.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhoai-verify-model-connected_{context}"]
+= Verifying that your AI model is accessible
+
+Before querying the model through the API, you can check to be certain that the model is accessible and ready to provide answers based on the connected data. The following examples continue with the {ovms}.
+
+.Prerequisites
+
+* You configured the AI model-serving runtime.
+* You uploaded your AI model to {microshift-short}.
+* The {microshift-short} cluster is running.
+* You installed `oc`.
+
+.Procedure
+
+. Get the IP address of the {microshift-short} cluster and assign it to the `IP` variable as the following example shows:
++
+[source,terminal]
+----
+$ IP=$(oc get nodes -o json | jq -r '.items[0].status.addresses[0].address')
+----
+
+. Identify the name of the route you created by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get route -n ai-test _<route-name>_ -o yaml <1>
+----
+<1> Replace <route_name> with the actual name of your route.
```

Comment:
Should this also be italicized? And I think it should be in backticks either way:
```suggestion
<1> Replace `_<route_name>_` with the actual name of your route.
```